libraries/rush-lib/src/logic/ProjectWatcher.ts

```diff
@@ -82,15 +82,23 @@ export class ProjectWatcher {
     const previousState: ProjectChangeAnalyzer = initialChangeResult.state;
     const repoRoot: string = Path.convertToSlashes(this._rushConfiguration.rushJsonFolder);
 
-    const pathsToWatch: Set<string> = new Set();
+    const pathsToWatch: Map<string, boolean> = new Map();
```

Comment:
Add a comment describing what is being mapped here. It's clear below but still good for readability